packages/remix-server-runtime/dev.ts

```diff
@@ -25,3 +25,22 @@ export async function broadcastDevReady(build: ServerBuild, origin?: string) {
 export function logDevReady(build: ServerBuild) {
   console.log(`[REMIX DEV] ${build.assets.version} ready`);
 }
+
+type DevServerRuntime = {
```

Comment:
Naming-wise, this doesn't feel like a runtime. What about something like:

```suggestion
type DevServerHooks = {
```